ts/client/src/accounts/mangoAccount.ts

```diff
@@ -231,9 +231,24 @@ export class MangoAccount {
    * @param bank
    * @returns native balance for a token, is signed
    */
-  public getTokenBalance(bank: Bank): I80F48 {
+  public getTokenBalance(group: Group, bank: Bank): I80F48 {
     const tp = this.getToken(bank.tokenIndex);
-    return tp ? tp.balance(bank) : ZERO_I80F48();
+    if (tp) {
+      const bal = tp.balance(bank);
+      for (const serum3Market of Array.from(
+        group.serum3MarketsMapByMarketIndex.values(),
```

Comment:
my understanding is that `values()` gives you and iterator, so making an Array out of it just for iterating should be redundant